backend/app.py

```
@@ -54,7 +58,8 @@ def process_query():
         return jsonify(result)
 
     except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)